src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { PropsInfo } from '../TablePeople/PropsInfo';
+
+export const PersonInfo: React.FC<PropsInfo> = ({
+  person, selectedSlug = '',
+}) => {
+  const {
+    sex, born, died, motherName, fatherName, mother, slug, father,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      // eslint-disable-next-line max-len
+      className={classNames({ 'has-background-warning': selectedSlug === slug })}
+    >
+      <td>
+        <PersonLink person={person} />
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      {mother
+        ? (
+          <td>
+            <PersonLink person={mother} />
+          </td>
+        )
+        : (
+          <td>
+            {motherName || '-'}
+          </td>
```

Comment:
Oh, i need it, because it was required in the task 👀
![image](https://user-images.githubusercontent.com/116097498/221971823-c396ba9a-b066-4d1f-b722-1951a09b31e8.png)